yb-voyager/cmd/importSchema.go

```diff
@@ -90,26 +95,10 @@ func importSchema() {
 	log.Infof("List of schema objects to import: %v", objectList)
 
 	// Import ALTER TABLE statements from sequence.sql only after importing everything else
-	isAlterStatement := func(objType, stmt string) bool {
-		stmt = strings.ToUpper(strings.TrimSpace(stmt))
-		return objType == "SEQUENCE" && strings.HasPrefix(stmt, "ALTER TABLE")
-	}
-
 	skipFn := isAlterStatement
 	importSchemaInternal(exportDir, objectList, skipFn)
-
-	// Import the skipped ALTER TABLE statements from sequence.sql if it exists
-	if slices.Contains(objectList, "SEQUENCE") {
-		skipFn = func(objType, stmt string) bool {
-			return !isAlterStatement(objType, stmt)
-		}
-		sequenceFilePath := utils.GetObjectFilePath(filepath.Join(exportDir, "schema"), "SEQUENCE")
-		if utils.FileOrFolderExists(sequenceFilePath) {
-			fmt.Printf("\nImporting ALTER TABLE DDLs from %q\n\n", sequenceFilePath)
-			executeSqlFile(sequenceFilePath, "SEQUENCE", skipFn)
-		}
```

Comment:
This is the problematic part. We should use the `importSchemaInternal()` instead of copying its behaviour here.